receiver/sqlserverreceiver/queries_test.go

```diff
@@ -108,3 +108,32 @@ func TestQueryTextAndPlanQueryContents(t *testing.T) {
 		})
 	}
 }
+
+func TestGetSQLServerQuerySamplesQuery(t *testing.T) {
+	queryTests := []struct {
+		name                     string
+		instanceName             string
+		getQuery                 func(uint64) string
+		expectedQueryValFilename string
+		maxResultPerQuery        uint64
```

Comment:
There are 10 remaining references to `maxResultPerQuery`, can we update those names to be `maxRowsPerQuery` to match the new config option name?